analyses/metrics/determine_qc_metric_schema.ipynb

```diff
@@ -0,0 +1,695 @@
+      "reads_per_molecule\n",
+      "reads_per_fragment\n",
+      "fragments_per_molecule\n",
+      "fragments_with_single_read_evidence\n",
+      "molecules_with_single_read_evidence\n",
+      "perfect_cell_barcodes\n",
+      "reads_mapped_intergenic\n",
+      "reads_unmapped\n",
+      "reads_mapped_too_many_loci\n",
+      "cell_barcode_fraction_bases_above_30_variance\n",
+      "cell_barcode_fraction_bases_above_30_mean\n",
+      "n_genes\n",
+      "genes_detected_multiple_observations\n",
+      "\n",
+      "gene metrics\n",
+      "\n",
+      "n_reads\n",
+      "noise_reads\n",
+      "perfect_molecule_barcodes\n",
+      "reads_mapped_exonic\n",
+      "reads_mapped_intronic\n",
+      "reads_mapped_utr\n",
+      "reads_mapped_uniquely\n",
+      "reads_mapped_multiple\n",
+      "duplicate_reads\n",
+      "spliced_reads\n",
+      "antisense_reads\n",
+      "molecule_barcode_fraction_bases_above_30_mean\n",
+      "molecule_barcode_fraction_bases_above_30_variance\n",
+      "genomic_reads_fraction_bases_quality_above_30_mean\n",
+      "genomic_reads_fraction_bases_quality_above_30_variance\n",
+      "genomic_read_quality_mean\n",
+      "genomic_read_quality_variance\n",
+      "n_molecules\n",
+      "n_fragments\n",
+      "reads_per_molecule\n",
+      "reads_per_fragment\n",
+      "fragments_per_molecule\n",
+      "fragments_with_single_read_evidence\n",
+      "molecules_with_single_read_evidence\n",
+      "number_cells_detected_multiple\n",
+      "number_cells_expressing\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "optimus_metric_dir = os.path.expanduser('~/Desktop/optimus')\n",
+    "\n",
+    "print('cell metrics\\n')\n",
+    "for c in pd.read_csv(os.path.join(optimus_metric_dir, 'merged-cell-metrics.csv.gz')).columns[1:]:\n",
+    "    print(c)\n",
+    "\n",
+    "print('\\ngene metrics\\n')\n",
+    "for c in pd.read_csv(os.path.join(optimus_metric_dir, 'merged-gene-metrics.csv.gz')).columns[1:]:\n",
+    "    print(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Schema\n",
+    "\n",
+    "Picard does not appear to follow a consistent schema, although one can map their results into JSON using Crimson, which is helpful. The matrix service will require the data be shipped in the same storage format recommend for sequencing data. Currently this appears to be Zarr. \n",
+    "\n",
+    "## ACs for ticket humancellatlas/secondary-analysis#105: \n",
+    "1. List of metrics to be updated: \n",
+    "    1. All, both pipelines. This is a small epic that will span multiple tickets to implement in a forward-compatible fashion.\n",
+    "2. names to be changed to:\n",
+    "    2. N/A, although some of the picard metrics will need to change by nature of their specificity to the files that generated them. For example, \"REF_BASE\" and \"ALT_BASE\" lose substantial meaning when combined into a columnar file. \n",
```

Comment:
Can you clarify what metrics need to be updated, if any? I see you listed out all of the metrics for SS2 and Optimus, but why would the A/C for renaming the metrics be N/A?